distribution/lib/Standard/Table/0.1.0/src/Data/Table.enso

```diff
@@ -699,6 +700,77 @@ type Table
         mask = OrderBuilder.buildReversedMask this.row_count
         Table <| this.java_table.applyMask mask
 
+    ## UNSTABLE
+
+       Returns a Text containing this table serialized to CSV.
+
+       Arguments:
+       - include_header: Specifies whether the first line of generated CSV
+         should contain the column names.
+       - always_quote: Specifies whether all fields in the resulting CSV should
+         be quoted. When this is set to `False`, only the fields containing the
+         `separator` in their contents will be quoted.
+       - separator: a sequence used to separate fields within a single row.
+       - line_ending: the style of line-endings to use in the generated CSV.
+
+       > Example
+         Convert a table to CSV.
+
+             import Standard.Examples
+
+             example_to_csv = Examples.inventory_table.to_csv
+    to_csv : Boolean -> Boolean -> Text -> Line_Ending_Style.Line_Ending_Style -> Text
+    to_csv include_header=True always_quote=False separator=',' line_ending=Line_Ending_Style.Unix =
+        Csv_Writer.writeString this.java_table include_header always_quote line_ending.to_text separator .to_csv_field
+
+    ## UNSTABLE
+
+       Serializes this table into CSV and writes it to the specified file.
+
+       Arguments:
+       - file: the file to write the generated CSV contents to. Note that other
+         files may be created or written to if `max_rows_per_file` is used.
```

Comment:
What happens if the file exists and has contents? You should probably have a `should_overwrite` parameter.

Reply:
We don't do it for any other file-writing method though? So this seems consistent and if we want to change it, we should probably change all places where this could happen. What do you think?

Reply:
Then we probably should have it elsewhere, but not worth it for now. File an issue.